packages/components/button/src/Button.stories.tsx

```diff
@@ -64,7 +66,7 @@ export const Design: StoryFn = _args => (
 )
 
 export const Intent: StoryFn = _args => (
-  <div className="gap-md flex flex-row">
+  <div className="gap-md flex flex-wrap">
```

Comment:
I have a warning in my editor that `flex-wrap` is deprecated and we should use `wrap` simply :)